### ISSUE TITLE
Glob pattern ignores are not working as expected

### ISSUE THREAD

**Issue by 5j9** [snoack/flake8-per-file-ignores#2]
What is the expected output of running flake8 in the test directory? I'm getting

```
a@D MINGW64 /d/flake8-per-file-ignores/test (master)
$ flake8.exe
.\bar.py:1:1: D100 Missing docstring in public module
.\foo.py:0:1: X100 Superfluous per-file-ignores for E302,F401
.\subdir\a\foo_a.py:1:1: F821 undefined name 'x'
.\subdir\a\foo_a.py:1:1: D100 Missing docstring in public module
.\subdir\b\foo_b.py:1:1: D100 Missing docstring in public module
.\subdir\b\foo_b.py:1:6: E712 comparison to True should be 'if cond is True:' or 'if cond:'
```

But according to the `setup.cfg` file I believe E712 and F821 should have been  ignored somehow. Am I missing something?

**Comment by snoack** [comment]
`test/setup.cfg` is using slashes as directory separator. However, on Windows, backslashes are expected. I just fixed this, by converting slashes to the path separator used by the current operating system. With the latest version from the `master` branch in Git, you should get following output, now:

    .\foo.py:0:1: X100 Superfluous per-file-ignores for E302,F401

**Comment by snoack** [comment]
I just released version 0.4 which only includes this fix.
